datacenter/ucp/2.1/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -34,19 +45,22 @@ be disconnected.
 Additionally, if UCP is not configured for high availability, you will be
 temporarily unable to:
 
-* Log in to the UCP Web UI
-* Perform CLI operations using existing client bundles
+* Log in to the UCP Web UI.
+* Perform CLI operations using existing client bundles.
 
 To minimize the impact of the backup policy on your business, you should:
 
 * Configure UCP for high availability. This allows load-balancing user requests
 across multiple UCP manager nodes.
 * Schedule the backup to take place outside business hours.
 
-## Backup command
+## Backup the swarm
+
+Start by [making a backup of your Docker Swarm](engine/swarm/admin_guide.md/#back-up-the-swarm).
+
+## Backup UCP
 
-The example below shows how to create a backup of a UCP manager node and
-verify its contents:
+Once you have a backup of your swarm, you can then create a backup of UCP:
 
 ```none
 # Create a backup and store it on /tmp/backup.tar
```

Comment:
I really don't like how it looks to put comments in CLI examples. It can confuse the reader who might think it's a root prompt. Maybe put it in the text before the example: "This example stores the backup in `/tmp/backup.tar`."